src/Neo/SmartContract/Native/RoleManagement.cs

```diff
@@ -78,7 +78,6 @@ private void DesignateAsRole(ApplicationEngine engine, Role role, ECPoint[] node
             list.AddRange(nodes);
             list.Sort();
             engine.SnapshotCache.Add(key, new StorageItem(list));
-
```

Comment:
please stop requiring me to update on unnecessary empty line changes, its automatically done by the dotnet format and not affect any functionality, if you like to change, just update it with github directly.

Reply:
Sorry I did it in github directly :S, my bad

Reply:
> Sorry I did it in github directly :S, my bad

nonono bro, you can update the pr at your free will, give suggestions, commit suggestions, remove empty lines that is not necessary, add empty lines to make the code easier to read,  that is all fine to me. I welcome any type of update to my pr personally. I am just saying should not require me to update empty lines.